harmony_service_lib/message.py

```diff
@@ -582,9 +582,7 @@ class Message(JsonObject):
     version : string
         The semantic version of the Harmony message contained in the provided JSON
     callback : string
-        The URL that services must POST to when their execution is complete.  Services
-        should use the `completed_with_*` methods of a Harmony Adapter to perform
-        callbacks to ensure compatibility, rather than directly using this URL
+        (Deprecated) The URL that services must POST to when their execution is complete.
```

Comment:
Is there any reason not to drop this field? Is anything still  using it? I only see it in tests.

Reply:
It requires changes to the data operation schema (affecting the main harmony repo as well). For this PR we're only removing code that can be safely removed without any coordination from other repos required.